config/environments/development.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.configure do
+  config.hosts.clear
```

Comment:
remove this line.
set tld_length equal to 0, and use localhost instead

`config.action_dispatch.tld_length = 0`

Reply:
always add new lines at the end of file with proper comment